styles.css

```diff
@@ -0,0 +1,595 @@
+  width: 100%;
+  margin: 0 auto;
+}
+
+.container-article {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.second-main h1 {
+  font-size: 20px;
+  font-family: georgia, "times new roman", times, serif;
+}
+
+.wrapper {
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(4, 155px);
+}
+
+.description p {
+  margin-right: 20px;
+}
+
+.box1 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 1;
+  grid-row-end: 2;
+}
+
+.box2 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 2;
+  grid-row-end: 3;
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.box3 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 3;
+  grid-row-end: 4;
+}
+
+.box4 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 4;
+  grid-row-end: 5;
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.second-footer {
+  margin-top: 50px;
+}
+
+.second-footer::before {
+  width: 60px;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  margin: 0 0 13px;
+  background-color: #000;
+}
+
+.second-footer p {
+  margin-bottom: 100px;
+  font-size: 13px;
+}
+
+.video-container {
+  width: 100%;
+  height: 0;
+  padding-bottom: 56.25%;
+  position: relative;
+  z-index: 10;
+}
+
+.video-container iframe {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  border: none;
+}
+
+.bottom-of-article {
+  font-size: 12px;
+  line-height: 16px;
+  color: rgb(102, 102, 102);
+}
+
+.bottom-button button {
+  width: 330px;
+}
+
+/* -------------------- THE MORE ARTICLES DIV ------------------- */
+
+.more-articles {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.left-div {
+  border-top: 1px solid #000;
+  width: 75%;
+  margin: 0 auto;
+  float: left;
+}
+
+.left-div h2 {
+  font-size: 16px;
+  margin-top: 20px;
+}
+
+.left-div a:visited {
+  color: #000;
+}
+
+.left-div a:hover {
+  text-decoration: underline;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.flex-item {
+  width: 272px;
+  height: 320px;
+  margin-top: 15px;
+  margin-right: 15px;
+}
+
+.bottom-img-div {
+  width: 272px;
+  height: 181px;
+}
+
+.flex-item img {
+  width: 100%;
+}
+
+.more-articles figcaption {
+  font-size: 10px;
+  line-height: 12.3px;
+  text-align: right;
+  margin: 3px 0;
+  color: #999;
+}
+
+.more-articles time {
+  font-size: 10px;
+  line-height: 12.3px;
+  margin: 3px 0;
+  color: #999;
+}
+
+.flex-item a h3 {
+  font-weight: 400;
+}
+
+.right-div {
+  width: 24%;
+  float: right;
+  border-top: 1px solid #000;
+}
+
+.right-div h2 {
+  font-size: 16px;
+  margin-top: 20px;
+  margin-bottom: 10px;
+  text-decoration: none;
+}
+
+.right-div h2:visited {
+  color: #000;
+}
+
+.right-div h2:hover {
+  text-decoration: underline;
+}
+
+/* ----------------------- BOTTOM NAVBAR --------------------- */
+
+.bottom-nav {
+  display: inline-block;
+  width: 100%;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)